data/i18n/hi/hi.toml

```diff
@@ -1,195 +1,195 @@
 # i18n strings for the English (main) site.
 
 [deprecation_warning]
-other = " documentation is no longer actively maintained. The version you are currently viewing is a static snapshot. For up-to-date documentation, see the "
+other = " प्रलेखन अब सक्रिय रूप से बनाए रखा नहीं है। वर्तमान में आप जो संस्करण देख रहे हैं वह एक स्थिर स्नैपशॉट है। अप-टू-डेट प्रलेखन के लिए, देखें "
 
 [deprecation_file_warning]
-other = "Deprecated"
+other = "पदावनत"
```

Comment:
```suggestion
other = "डेप्रिकॉटेड"
```

Reply:
`पदावनत` is not a word many Hindi speakers understand, better we just transliterate it